### ISSUE TITLE
bug: uint and uint256 parsed different in  EIP-712 Typed Data event signatures

### ISSUE THREAD

**Issue by krzkaczor** [wagmi-dev/viem#166]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

0.1.7

### Current Behavior

This results in two different types ie. `uint` doesn't get coerced to `uint256`.
```
// works
  const logs = await client.getLogs({
    address: erc20Address,
    event: parseAbiItem('event Transfer(address indexed from, address indexed to, uint256 value)'),
    fromBlock: 16790000n,
  })

//doesnt work (no results)
  const logs2 = await client.getLogs({
    address: erc20Address,
    event: parseAbiItem('event Transfer(address indexed from, address indexed to, uint value)'), // uint instead of uint256 here
    fromBlock: 16790000n,
  })
```

Maybe this issue should be moved to abitype, i am not sure which library should be responsible for ABI normalization.

### Expected Behavior

Both queries works

### Steps To Reproduce

Just run the code stored above. 

### Link to Minimal Reproducible Example (StackBlitz, CodeSandbox, GitHub repo etc.)

https://stackblitz.com/edit/viem-getting-started-vwxiwl?file=index.ts

### Anything else?

Sorry I couldn't create repro on stackblitz due to some random error. I think the problem is also that template `new.viev.sh ` uses old viem version in package.json.

Anyway, great job on library! I can't wait to use it more.

**Comment by greenlucid** [comment]
Isn't this intended? uint is a solidity alias, is it ABI conforming?

**Comment by krzkaczor** [comment]
@greenlucid you are right that EIP-712 doesn't support `uint` alias (i copied the whole selector from the sourcecode -- that's why i hit that). 

However, I think that there should be some kind of error instead of a silent failure.
